packages/mobile/react-native.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  dependencies: {
+    'react-native-confirm-device-credentials': {
```

Comment:
This prevents autolinking because we only want to add this in devices over api 23